elastic/kibana
# [Lens] Fix permission problem for "Save and return" button

Fixes https://github.com/elastic/kibana/issues/127800

By only checking for "isSaveable" for "Save and return" logic, not for dashboard capabilities (as dashboard isn't the only app integrating with Lens this way).

To test:
* Create new user which doesn't have access to dashboards
* Go to cases
* Lens integration should work
